src/js/components/Markdown/README.md

```diff
@@ -1,6 +1,9 @@
 ## GrommetMarkdown
 Markdown formatting using Grommet components.
 
+Grommet uses 'markdown-to-jsx' in Markdown component,
+      you can see all the options in the documentation.
```

Comment:
thanks for adding this, can you also add the new prop of `options` (similarly to `components` prop) and direct to 'markdown-to-jsx' with a link?

Reply:
Yes, sure!